test/unit/utility/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+#
+# Distributed Linear Algebra with Future (DLAF)
+#
+# Copyright (c) 2018-2019, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+add_subdirectory(mc)
```

Comment:
We are adding a new file, 10 lines of license information for 1 line of code. Wouldn't it be simpler to add the folder from the top CMakeLists.txt file?